code/src/java/pcgen/output/actor/FactKeyActor.java

```diff
@@ -40,6 +42,9 @@
  */
 public class FactKeyActor<T> implements OutputActor<CDOMObject>
 {
+	private static final ObjectWrapperFacet WRAPPER_FACET = FacetLibrary
```

Comment:
FacetLibrary.getFacet is "self-initializing" in that if the items is not already known, it is requested from Spring... so this usage is not an order of operations problem.